app.config.js

```diff
@@ -1,17 +1,17 @@
 let api;
 
 switch(process.env.NODE_ENV) {
-case 'production':
-  api = "http://172.16.117.140";
-  break;
-case 'development':
-  api = "http://172.16.117.140";
-  break;
-case 'test':
-  api = "http://mocked-api-path";
-  break;
-default:
-  api = "http://172.16.117.140";
+  case 'production':
+    api = "https://api-test.cacophony.org.nz";
+    break;
+  case 'development':
+    api = "https://api-test.cacophony.org.nz";
```

Comment:
Copy, will change it. Ideally we won't have a config file committed though? I put it back as travis needs one as it's now configured (I'm guessing.)

Reply:
I think this is why Travis is now complaining at the lack of config file. ie the config imports used to be marked eslint-disable-line but are no longer, hence Travis wants the config file to exist.

![eslint issue](https://user-images.githubusercontent.com/24260476/43927294-2a0c84a6-9c81-11e8-9f3e-4899bed8450d.png)

Reply:
The eslint-disable-line was there because of a mis-config in the babelrc (which I've since fixed) but yes, side effect was a happy travis.

I've changed to http://localhost:1080 for now ..